templates/release_checklist.md

```diff
@@ -0,0 +1,149 @@
+
+- [ ] Clone or update the [release toolkit](https://href.li/?https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile) and review the [release script instructions](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md).
+- [ ] [Notifiy open Gutenberg Mobile PRs assigned to {{version_number}} milestone]({{milestone_url}}).
+<details>
+<summary>Message Template</summary>
+<p>
+
+```markdown
+Hello!
+We will cut the {{version_number}} release on {{release_date}}.
+I plan to circle back and bump this PR to the next milestone then,
+but please let me know if you’d rather us work to include this PR in {{version_number}}.
+Thanks!
+```
+
+</p>
+</details>
+
+- [ ] Create WPiOS and WPAndroid builds from `develop`.
+- [ ] [Perform general writing flow tests on Android](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
+- [ ] [Perform general writing flow tests on iOS](https://github.com/wordpress-mobile/test-cases/tree/master/test-cases/gutenberg/writing-flow).
+
+<!-- /scheduled_release_only -->
+
+## Create the Release ({{release_date}})
+
+<!-- scheduled_release_only -->
+
+On {{release_date}}:
+
+<!-- /scheduled_release_only -->
+
+- [ ] [Verify Aztec is up to date](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md#handling-aztec-updates).
+- [ ] Run `./release_automation.sh` from the release toolkit to generate the main apps integration PRs [\*](#).
+- [ ] Notify team that the release has been cut.
+<details>
+<summary>Message Template</summary>
+<p>
+
+```markdown
+⚠️ The gutenberg-mobile {{version_number}} release branches are now cut.
+Please do not merge any Gutenberg-related changes into the WPiOS or WPAndroid
+develop branches until after the main apps cut their own releases next week.
+If you’d like to merge changes now, merge them into the `gutenberg/after_{{version_number}}` branches.
+```
+
+</p>
+</details>
+
+- [ ] [Verify localization string files have been generated properly](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/develop/Releasing.md).
+- [ ] Update `Unreleased` section in [`RELEASE-NOTES.txt`](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) and [`gutenberg/../CHANGELOG.md`](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/CHANGELOG.md). Add a new `Unreleased` section after updating the release notes.
+- [ ] Perform quick smoke test of the editor on both iOS and Android builds
+- [ ] Update [Gutenberg Mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pulls?q=is%3Apr+is%3Aopen+label%3Arelease-process) description. Make sure the "Changes" section informs testers on what is in the release.
+- [ ] Mark all 4 PRs ready for review.
+
+<!-- non_scheduled_release_only -->
+
+- [ ] Notify the team members directly resposible for handling the releases of WPiOS and WPAndroid.
```

Comment:
It would be great if we include a message template for this, it could be something like:

> Hi team. I wanted to provide a heads up that the Gutenberg Mobile team is starting work on the {{version_number}} beta release into the {{main_apps_version_number}} release branches. We'll provide an update once the integration is complete. Let us know if you have any questions or concerns. Thanks!

**NOTE:** For this message we would need to ask the version number of the apps.